src/GitVersionExe.Tests/ExecCmdLineArgumentTest.cs

```diff
@@ -98,7 +98,7 @@ public void WorkingDirectoryDoesNotExistCrashesWithInformativeMessage()
                 args,
                 PathHelper.GetCurrentDirectory());
 
-            exitCode.ShouldBe(0);
+            exitCode.ShouldBe(1);
```

Comment:
I could also change this to the following to match the code before switching to use generic hosting:
```c#
exitCode.ShouldNotBe(0);
```